vault.gemspec

```diff
@@ -19,6 +19,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "net-http-persistent", "~> 3.0"
```

Comment:
I really really really tried to have zero dependencies in the gem. Could we vendor this or implement it ourselves?